.config/jest-setup.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * ⚠️⚠️⚠️ THIS FILE WAS SCAFFOLDED BY `@grafana/create-plugin`. DO NOT EDIT THIS FILE DIRECTLY. ⚠️⚠️⚠️
+ *
+ * In order to extend the configuration follow the steps in .config/README.md
+ */
+
+import '@testing-library/jest-dom';
```

Comment:
The file `src/tests/setupTests.ts` can be deleted. Looks like this is doing what that file is already doing.

Reply:
done! and sorry about the force-push, i wasn't sure how else to fix the yarn.lock issue nicely.